ui-tests/tests/jupyterlab_eduhelx_submission.spec.ts

```diff
@@ -16,6 +16,6 @@ test('should emit an activation console message', async ({ page }) => {
   await page.goto();
 
   expect(
-    logs.filter(s => s === 'JupyterLab extension jupyterlab_eduhelx_submission is activated!')
+    logs.filter(s => s === 'JupyterLab extension eduhelx_jupyterlab_student is activated!')
```

Comment:
You didn't change the name of this file

Reply:
Mb, I was lazy and didn't realize vscode search doesn't grep filenames. Fixing all the filenames now and another review to make sure I'm not missing stuff.